module/VuFind/src/VuFind/RecordDriver/SolrOverdrive.php

```diff
@@ -250,8 +251,9 @@ public function getPreviewLinks()
      */
     public function supportsAjaxStatus()
     {
-        // TODO: add this as an Overdrive configuration to turn it off
-        return true;
+        $this->debug('ajax status:' . $this->config->enableAjaxStatus);
+        $this->debug('all configs:' . print_r($this->config, true));
```

Comment:
Do we need all these debug outputs here?

Reply:
No. Should I generally go through all the comments and trim it down to the most useful?

Reply:
It wouldn't hurt. As a rule of thumb: if it's going to help somebody who's having trouble with Overdrive, feel free to leave it. If it's something you needed during development but no longer expect to require, get rid of it. :-)

Reply:
Remove quite a few debug statements